sys/kern/imgact_elf.c

```diff
@@ -527,9 +526,72 @@ __elfN(check_header)(const Elf_Ehdr *hdr)
 	return (0);
 }
 
+#if __has_feature(capabilities)
+/*
+ * Create a reservation for the image activator to map the executable
+ * image into.
+ */
+static int
+__elfN(build_imgact_capability)(struct image_params *imgp,
+    void * __capability *imgact_cap, const Elf_Ehdr *hdr, const Elf_Phdr *phdr,
+    u_long rbase)
```

Comment:
```suggestion
    u_long *rbase)
```
To allow updating it when the reservation is created.